downloader/src/main/kotlin/Main.kt

```diff
@@ -86,6 +87,7 @@ object Main {
      * [vcsInfo] is set to a non-null value.
      */
     data class DownloadResult(
+        val dateTime: Instant,
```

Comment:
Could you please add `dateTime` also to the docs above, and while at it, maybe also `downloadDirectory`, so we have all properties documented? In paricular, is `dateTime` the time when the download started or finished?

Reply:
Done. It was the time the download finished, but I have changed it to the start time because this seemed more relevant to me.